package.json

```diff
@@ -41,10 +41,9 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.2",
     "@fortawesome/free-solid-svg-icons": "^5.2.0",
     "@fortawesome/react-fontawesome": "^0.1.0",
-    "aframe": "github:mozillareality/aframe#afa764e5f515784e7349f834a53142bcadcd0ffe",
+    "aframe": "github:mozillareality/aframe#remove-tracked-controls",
```

Comment:
Will change this to point to `aframe#hubs/master` after merging https://github.com/MozillaReality/aframe/pull/8